app/views/works/_meta.html.erb

```diff
@@ -56,11 +56,23 @@
       </dd>
     <% end %>
 
-    <dt class="stats"><%= ts("Stats") %>:</dt>
+    <dt class="stats"><%= ts("Stats:") %></dt>
     <dd class="stats">
 <% end %><!-- end of cache -->
 
       <%= work_meta_list(@work, @chapter) %>
-    </dd>
+      </dd>
+      <% if logged_in_as_admin? %>
+        <dt class="ip">
+          <%= ts('IP Address:') %>
+        </dt>
+        <dd class="ip">
```

Comment:
I took a look because you mentioned CSS problems, and it looks okay to me:
![ip_address_4211](https://cloud.githubusercontent.com/assets/907055/5655782/8d1373c4-96a0-11e4-847d-2ec940cb4fc8.png)

Which browser are you having a problem with, or what do you want it to look like?

Also, the indenting here is still a bit wrong. The if/else/end is inside the dd, so everything belonging to it needs to move over two more spaces:
```
<dd class="ip">
  <% if @work.ip_address.nil? %>
    <%= ts('No address recorded') %>
  <% else %>
    <%= @work.ip_address %>
  <% end %>
</dd>
```